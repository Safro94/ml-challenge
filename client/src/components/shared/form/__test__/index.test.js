import React from 'react';
import { render } from '@testing-library/react';

import Form from '..';

describe('Form', () => {
	it('should render OK', () => {
		//Act
		const { container } = render(
			<Form>
				<h1>Test</h1>
			</Form>
		);

		//Assert
		expect(container).toMatchSnapshot();
	});
});
