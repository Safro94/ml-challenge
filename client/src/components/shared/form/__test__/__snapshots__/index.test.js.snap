// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Form should render OK 1`] = `
<div>
  <form>
    <h1>
      Test
    </h1>
  </form>
</div>
`;
